bucket/vulkan.json

```
@@ -10,9 +10,16 @@
         "Make sure you have the vulkan driver installed.",
         "Variety of drivers of different graphic card vendors could be found at the bottom of 'https://www.khronos.org/vulkan/'"
     ],
-    "url": "https://sdk.lunarg.com/sdk/download/1.2.154.1/windows/VulkanSDK-1.2.154.1-Installer.exe#/dl.7z",
-    "hash": "b64471f3a720e649c1fae6535ea83b8c642655ebed1485bfdf15bf4d88f746d9",
+    "url": [
+        "https://sdk.lunarg.com/sdk/download/1.2.154.1/windows/VulkanSDK-1.2.154.1-Installer.exe#/dl.7z",
+        "https://raw.githubusercontent.com/rgozim/Main/master/scripts/vulkan/install-debug-libs.ps1"
+    ],
+    "hash": [
+        "b64471f3a720e649c1fae6535ea83b8c642655ebed1485bfdf15bf4d88f746d9",
+        "b69750aa36e35483981fc4f47f156f45036073d1f9d0325f2747d2fe9fb07d8f"
+    ],
     "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstal*\" -Recurse",
+    "post_install": "& \"$dir\\install-debug-libs.ps1\" $cachedir $dir $version",
```

I haven’t tested it, but how about trying:
```suggestion
    "post_install": "& \"$bucketsdir\\main\\scripts\\vulkan\\install-debug-libs.ps1\" $cachedir $dir $version",
```
per https://github.com/lukesampson/scoop/wiki/Pre--and-Post-install-scripts#variables, and skip downloading
https://raw.githubusercontent.com/rgozim/Main/master/scripts/vulkan/install-debug-libs.ps1
since it is already available in the local main bucket.

---

Is using a local .ps1 "allowed"? Other examples, such as the [Jetbrains bucket](https://github.com/Ash258/Scoop-JetBrains/blob/master/bucket/CLion-EAP-portable.json), seem to take the approach of downloading their scripts.

---

Yes, of course. I think we need to move away from downloading files that are available locally. Note that the URL to this script is pointing to your fork, and not to the main bucket.

---

Using `$bucketsdir\\main\\scripts\\vulkan` in place of just `$dir\\` doesn't allow for a custom bucket name, as far as I can see. Would this not be an issue or  is there anyway for the `post_install` script to infer the directory it's bucket is running from?